src/lepton/uncompressed_components.hh

```diff
@@ -271,6 +271,9 @@ public:
     
     void reset() {
         bit_progress_ -= bit_progress_;
+        for (int cmp = 0; cmp < (int)header_.size() && cmp < cmpc_; cmp++) {
```

Comment:
`.size()` returns `uint32_t`
`.operator[]` receives `uint32_t`
May be we can replace `int` to `uint32_t` ?
Here and above.

Reply:
I just copied this snippet intact. AFAIR cmpc_ is the number of color planes i.e. usually 3

Reply:
Ok, it's only suggestion, not requirement. Up to you.